.github/workflows/build-starter-templates.yml

```diff
@@ -19,46 +20,65 @@ jobs:
           - tinasaurus
           - basic
         node-version: [18, 20, 22]
+    outputs:
+      failed: ${{ steps.report-errors.outputs.failed }}
 
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
 
       - uses: pnpm/action-setup@v4
         name: Install pnpm
-        if: ${{ matrix.package-manager == pnpm }}
-        id: pnpm-install
         with:
-          package_json_file: package.json
-          run_install: false
+          version: 9
```

Comment:
do we have docs/create-tina-app checks for the pnpm version?

Reply:
We do not. I will go ahead and update tina.io to include information about the supported package managers.